catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrFilterDelegate.java

```diff
@@ -82,6 +92,21 @@
 
     private static final WKTWriter WKT_WRITER = new WKTWriter();
 
+    // For queries we use repairConvexHull which my cause false positives to be returned but this
```

Comment:
Added a note to the Catalog UI search documentation since that seems like the most likely place a user would run into this. 
![screen shot 2017-02-21 at 9 30 54 am](https://cloud.githubusercontent.com/assets/5248090/23174332/3d3fee54-f819-11e6-831e-c9dd99b62417.png)